pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -502,9 +511,53 @@ func shouldIgnore(a admission.Attributes) (bool, error) {
 		return true, nil
 	}
 
+	// if this is an update, see if we are only adding annotations that should be ignored during admission
+	// for example, the OVN controller adds networking annotations that shouldn't cause the pod to go through admission again
+	if a.GetOperation() == admission.Update && isOnlyAddingInformativeAnnotations(a.GetObject(), a.GetOldObject(), kapihelper.Semantic) {
+		return true, nil
+	}
```

Comment:
to avoid memory and computational complexity (both these methods DeepCopy the whole incoming object and compare them with the old ones), I'd propose something more like this:

```golang
if a.GetOperation() == admission.Update {
    oldPod, ok := a.GetOldObject().(*coreapi.Pod)
    if  !ok {
        return false, admission.NewForbidden(a, fmt.Errorf("well I've seen just about everything now"))
    }

    if !kapihelper.Semantic.DeepEqual(newPod.Spec, oldPod.Spec) {
        return false, nil
    }
   
    if shouldIgnoreMetaChanges(newPod.ObjectMeta, oldPod.ObjectMeta) {
        return true, nil
    }
}
```